exchanges/okx/okx_websocket.go

```diff
@@ -287,7 +287,7 @@ func (e *Exchange) WsAuth(ctx context.Context) error {
 	e.Websocket.Wg.Add(1)
 	go e.wsReadData(ctx, e.Websocket.AuthConn)
 	e.Websocket.AuthConn.SetupPingHandler(request.Unset, websocket.PingHandler{
-		MessageType: gws.TextMessage,
+		MessageType: gws.PingMessage,
```

Comment:
This needs to be reverted as it fails to send a correct ping message for auth connection and will make it unstable. I have added it to the diff.